<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaperNet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainWindow.qrc">
    <normaloff>:/MainWindow/Images/ToiletPaper.png</normaloff>:/MainWindow/Images/ToiletPaper.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="margin">
     <number>5</number>
    </property>
    <item>
     <widget class="QTableView" name="tvPapers"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="ToolBar" name="toolBarMain">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOptions"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QToolBar" name="toolBarPapers">
   <property name="windowTitle">
    <string>Papers toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport"/>
   <addaction name="actionAddPaper"/>
   <addaction name="actionDelPaper"/>
  </widget>
  <widget class="QDockWidget" name="dockDetails">
   <property name="windowTitle">
    <string>Details</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Abstract</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="teAbstract">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Note</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="teNote">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Attachments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="AttachmentsWidget" name="widgetAttachments" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockTags">
   <property name="windowTitle">
    <string>Tags</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitterTags">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QListView" name="lvTags">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>All tags</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="btFilter">
           <property name="icon">
            <iconset resource="MainWindow.qrc">
             <normaloff>:/MainWindow/Images/Filter.png</normaloff>:/MainWindow/Images/Filter.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btAddTag">
           <property name="icon">
            <iconset resource="MainWindow.qrc">
             <normaloff>:/MainWindow/Images/Add.png</normaloff>:/MainWindow/Images/Add.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btDelTag">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="icon">
            <iconset resource="MainWindow.qrc">
             <normaloff>:/MainWindow/Images/Minus.png</normaloff>:/MainWindow/Images/Minus.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btAddTagToPaper">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="icon">
            <iconset resource="MainWindow.qrc">
             <normaloff>:/MainWindow/Images/Up.png</normaloff>:/MainWindow/Images/Up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="btDelTagFromPaper">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="icon">
            <iconset resource="MainWindow.qrc">
             <normaloff>:/MainWindow/Images/Down.png</normaloff>:/MainWindow/Images/Down.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QListView" name="lvAllTags">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockSnippets">
   <property name="windowTitle">
    <string>Snippets</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="SearchToolBar" name="toolBarSearch">
   <property name="windowTitle">
    <string>Search toobar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Images/Exit.png</normaloff>:/MainWindow/Images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Images/Options.png</normaloff>:/MainWindow/Images/Options.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Images/Import.png</normaloff>:/MainWindow/Images/Import.png</iconset>
   </property>
   <property name="text">
    <string>Import paper</string>
   </property>
   <property name="toolTip">
    <string>Import paper</string>
   </property>
  </action>
  <action name="actionAddPaper">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Images/AddFile.png</normaloff>:/MainWindow/Images/AddFile.png</iconset>
   </property>
   <property name="text">
    <string>Add paper</string>
   </property>
   <property name="toolTip">
    <string>Add paper</string>
   </property>
  </action>
  <action name="actionDelPaper">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/MainWindow/Images/Minus.png</normaloff>:/MainWindow/Images/Minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete paper</string>
   </property>
   <property name="toolTip">
    <string>Delete paper</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AttachmentsWidget</class>
   <extends>QWidget</extends>
   <header>AttachmentsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolBar</class>
   <extends>QToolBar</extends>
   <header>ToolBar.h</header>
  </customwidget>
  <customwidget>
   <class>SearchToolBar</class>
   <extends>QToolBar</extends>
   <header>SearchToolBar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
